spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/secretmanager/SecretManagerConfigDataLocationResolverUnitTests.java

```diff
@@ -90,4 +97,27 @@ void registerBean() {
     when(context.getBinder()).thenReturn(new Binder());
     when(context.getBootstrapContext()).thenReturn(defaultBootstrapContext);
   }
+
+  @Test
+  void testSecretManagerProjectIdTakesPrecedence() {
+    when(secretManagerProperties.getProjectId()).thenReturn("secret-manager-property-id");
+    GcpProjectIdProvider projectIdProvider =
```

Comment:
Could you also add a line to setup gcpProperties? To make sure the secret manager property is indeed returned first.

Reply:
done b269abf622e4e06fe1ea6e210bfd883f5f4a9bf5